internal/common/echo_logrus.go

```diff
@@ -0,0 +1,190 @@
+package common
+
+import (
+	"encoding/json"
+	"io"
+
+	"github.com/labstack/gommon/log"
+	"github.com/sirupsen/logrus"
+)
+
+// EchoLogrusLogger extend logrus.Logger
+type EchoLogrusLogger struct {
+	*logrus.Logger
+}
+
+var commonLogger = &EchoLogrusLogger{
+	Logger: logrus.New(),
```

Comment:
do we have to instantiate a  new logger? can't we just use the static one? Becuase this logger will hve to be configured as well.